samples/SampleApp/Program.cs

```diff
@@ -7,16 +7,9 @@ namespace SampleApp
 {
     public class Program
     {
-        private IServiceProvider _services;
-
-        public Program(IServiceProvider services)
-        {
-            _services = services;
-        }
-
-        public void Main(string[] args)
+        public static void Main(string[] args)
         {
-            new Microsoft.AspNet.Hosting.Program(_services).Main(new[] {
+            Microsoft.AspNet.Hosting.Program.Main(new[] {
                 "--server", "Microsoft.AspNet.Server.Kestrel"
             });
```

Comment:
It looks like there's an existing bug here where it discards the incoming args. You can concat those like it does in the other Program.cs.